pkg/pillar/cmd/zedmanager/updatestatus.go

```diff
@@ -395,15 +398,14 @@ func doInstall(ctx *zedmanagerContext,
 					log.Infof("Resolve status not found for %s", ss.ImageID)
 					ss.HasResolverRef = true
 					MaybeAddResolveConfig(ctx, ss)
-					// XXX state name could be "waiting for tag resolutions(s)" but
-					// painful to introduce a new state end to end
-					minState = types.INITIAL
-					ss.State = types.INITIAL
+					ss.State = types.RESOLVING_TAG
 					changed = true
 					continue
 				}
 				log.Infof("Processing ResolveStatus for storage status (%v) in app instance (%v)",
 					ss.ImageID, status.UUIDandVersion.UUID)
+				ss.State = types.RESOLVED_TAG
```

Comment:
I think ```RESOLVED_TAG``` state will never get published to the controller because it will be overwritten by other states before publishing the instance status.

Reply:
Yes, I think you are right. But I added it for completeness.

Reply:
Ok, thanks!